creator-node/src/routes/files.js

```diff
@@ -159,23 +160,18 @@ module.exports = function (app) {
       res.setHeader('Content-Disposition', contentDisposition(req.query.filename))
     }
 
-    // Fire-and-forget rehydration
-    rehydrateIpfsFromFsIfNecessary(
-      req,
-      CID,
-      queryResults.storagePath
-    )
-
+    // Attempt to stream file to client from fsß
     try {
-      // Attempt to stream file to client.
+      // Add a rehydration task to the queue to be processed in the background
+      await RehydrateIpfsQueue.addRehydrateIpfsFromFsIfNecessaryTask(CID, queryResults.storagePath, { logContext: req.logContext })
```

Comment:
I still feel that we should not be awaiting here -- there's no need. since we don't care about what happens, it just adds more time to each request that we don't want.

Reply:
we do care if it gets added to the queue though? the rehydrate part can be done async but don't we want to ensure that a task gets at least added to the queue?

Reply:
discussed offline - will remove await bc we're not using response